extension/src/NABfunctions.ts

```diff
@@ -1458,3 +1465,69 @@ function activeTextEditorIsXlf(): boolean {
     vscode.window.activeTextEditor.document.uri.fsPath.endsWith("xlf")
   );
 }
+
+export async function createPermissionSetForAllObjects(): Promise<void> {
+  logger.log("Running: createPermissionSetForAllObjects");
+  Telemetry.trackEvent("createPermissionSetForAllObjects");
+  try {
+    const appManifest = SettingsLoader.getAppManifest();
+    const settings = SettingsLoader.getSettings();
+    const allObjects = await WorkspaceFunctions.getAlObjectsFromCurrentWorkspace(
+      settings,
+      appManifest,
+      true,
+      false,
+      false
+    );
+    const firstNumber = appManifest.idRanges[0]
+      ? appManifest.idRanges[0].from
+      : 50000;
```

Comment:
I think that null coalesce would work here. But I'd say it's a matter of preference.
```suggestion
    const firstNumber = appManifest.idRanges[0].from ?? 50000;
```

Reply:
I will never get used to null coalesce! 😂

Reply:
fixed